Missionframework/modules/02_build/fnc/fn_build_displayScript.sqf

```diff
@@ -1,4 +1,3 @@
-#include "..\ui\defines.inc"
```

Comment:
This was needed here.

Script iterates over an array of IDCs that is in common defines.

[Here](https://github.com/KillahPotatoes/KP-Liberation/blob/7a19fb1d740555e597645dd40aad5506cc359ac0/Missionframework/modules/02_build/fnc/fn_build_displayScript.sqf#L80:L83).